pianoPlayer/main.js

```diff
@@ -0,0 +1,108 @@
+/* ISSUES & TODOs
+- add music!!
+- stop propagation / capture=true for when the keys are pressed (excluding the letters!)
+- mouseup event not caught when mouseout before
+- prevent key from overlapping the dark keys when it's being pressed
+- fix skeuomorphic design for upper keys
+*/
```

Comment:
I _just_ learning this word yesterday! (`skeuomorphic`) Nice!

And this is fine, though to borrow one of Zack's phrases, I'd like to plant the seed that these TODOs would, in the future, be small tickets/trello cards. That way, they'd be easier to find whenever you're done with one ticket and thinking something like "It's the end of my day, but still wanna do one or two small things. Like coding dessert". And you'd find one or two of on the board and knock them out.

For this project it'd be wild overkill, but like I said, just planting that seed.